src/components/DealInteraction.tsx

```diff
@@ -46,6 +68,26 @@ export const DealInteraction: FunctionComponent<DealInteractionProps> = ({
 							description: "Interact with deal: amount input placeholder",
 						})}
 						name="amount"
+						required={true}
+						rules={[
+							{
+								required: true,
+								message: intl.formatMessage({
+									defaultMessage: "'amount' is required",
+									description: "Deal interaction: amount required validation message",
+								}),
+							},
+							{
+								validator(_, value) {
+									return validateMinAmount(value);
+								},
+							},
+							{
+								validator(_, value) {
+									return validateMaxAmount(value);
```

Comment:
Can we also have the `max amount` for repay?

Reply:
there is, or do you mean that we should validate on the yet to be repaid part?
<img width="625" alt="image" src="https://user-images.githubusercontent.com/11614239/176139829-65527b8b-d0d4-48d9-b756-43f23510e05e.png">

Reply:
yes, I mean validate on yet to be repaid part.